applications/registration-react/src/pages/catalogs-page/catalogs/catalog-item/catalog-item.component.jsx

```diff
@@ -31,11 +31,9 @@ export const CatalogItem = props => {
         <a className="catalog-item" href={linkUri}>
           <div className={itemClass}>
             <h3 className={iconClass}>{localization.catalogs[type]}</h3>
-            {itemsCount && (
-              <span className="fdk-text-size-small fdk-color-neutral-dark">
-                {itemsCount} {localization.catalogs.type[type]}
-              </span>
-            )}
+            <span className="fdk-text-size-small fdk-color-neutral-dark">
+              {itemsCount || 0} {localization.catalogs.type[type]}
```

Comment:
egentlig er det 3 foskjellige senarioer her.
1) fikk svar (0 eller mer)
2) pending
3) api error

"pending" render er "kind of" løst nå med resolve - det mener hele knappen er borte, blir rendret litt senere.
men du løser error case også med 0.

jeg anbefaler å ikke render noe for error. Egentlig kunne det være kanskje litt elegant hvis du også fikser pending løsning - at det er ikke knapp som ble delayet, men bare den nummer. Så du kan bevege "resolve" en steg dypere.